CHANGELOG.asciidoc

```diff
@@ -78,6 +78,8 @@ https://github.com/elastic/beats/compare/v5.1.1...master[Check the HEAD diff]
 - Make system process metricset honor the cpu_ticks config option. {issue}3590[3590]
 - Support common.Time in mapstriface.toTime() {pull}3812[3812]
 - Fixing panic on prometheus collector when label has , {pull}3947[3947]
+- Fixing prometheus collector to aggregate metrics based on metric family. {pull}4075[4075]
```

Comment:
I would not put this under bugfixes but breaking changes as it changes the data structure.